### PR TITLE
Label control implementation

### PR DESCRIPTION
**Pull request by HackerDelphi**
## What does the pull request do?
Adds Label control similar to WPF's one

## What is the current behavior?
Have Label control in Avalonia. 
This can help to migrate WPF XAML files easier than before - Label is activelly used in WPF

## What is the updated/expected behavior with this PR?
Added Label control with Target property

## How was the solution implemented (if it's not obvious)?
Label control is inherited from ContentControl (like in WPF) and adds additional processing of Content to change ContentTemplate if no one is provided to use AccessText for hot key processing..

## Checklist
- [ ] Added unit tests (if possible)?
- [x] Added XML documentation to any related classes?
- [ x] Consider submitting a PR to https://github.com/AvaloniaUI/Avaloniaui.net with user documentation

## Breaking changes
None

## Fixed issues
None

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=4618) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=4618) before we can accept your contribution.<br/><br/>:x: HackerDelphi [sign now](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia?pullRequest=4618)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/AvaloniaUI/Avalonia?pullRequest=4618) it.</sub>

File: src/Avalonia.Controls/Label.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        private void LabelActivated()
+        {
+            Target?.Focus();
+        }
+
+        /// <summary>
+        /// Handler for Content property change event
+        /// </summary>
+        /// <param name="obj">new value</param>
+        private void ContentChanged(object obj)
```
Comment (kekekeks) [inline src/Avalonia.Controls/Label.cs:62]: @grokys 
It seems that WPF implements support for access keys directly in ContentPresenter. Should we port that behavior or just keep hacks like this one?

File: src/Avalonia.Themes.Default/Label.xaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<Style xmlns="https://github.com/avaloniaui" Selector="Label">
```
Comment (kekekeks) [inline src/Avalonia.Themes.Default/Label.xaml:1]: Template for the fluent theme is missing (you can copy-paste this one)

Comment (maxkatz6) [reply src/Avalonia.Themes.Default/Label.xaml:1]: PR targets 0.9.12 for some reason

**Comment by FoggyFinder** [comment]
Just curious: Why? Label always seemed like redundant control to me.

File: src/Avalonia.Controls/Label.cs (hunk 1 above)
Comment (HackerDelphi) [reply src/Avalonia.Controls/Label.cs]: > Just curious: Why? Label always seemed like redundant control to me.

It is very needed by users who works with keyboard/hotkeys. I know such users and their work contains of entering many documents with complex structure and ability of using hotkeys to travel through inputs is very important for them. 
They hate programs without hotkeys.

File: src/Avalonia.Themes.Default/Label.xaml (hunk 2 above)
Comment (HackerDelphi) [reply src/Avalonia.Themes.Default/Label.xaml:1]: > Template for the fluent theme is missing (you can copy-paste this one)

Where is it? can't find in Avalonia Repo... :( maybe i'm blind???? ;(

Comment (danwalmsley) [reply src/Avalonia.Themes.Default/Label.xaml:1]: https://github.com/AvaloniaUI/Avalonia/tree/master/src/Avalonia.Themes.Fluent

**Comment by worldbeater** [comment]
Probably worth changing the target branch from `AvaloniaUI:release/0.9.12` to `master`.

**Comment by kekekeks** [comment]
@FoggyFinder Label is supposed to be attached to a textbox, so it would activate said textbox on click.

**Comment by HackerDelphi** [comment]
1. Merged master to branch.
2. Added Fluent theme resources.

File: src/Avalonia.Themes.Default/Label.xaml (hunk 2 above)
Comment (HackerDelphi) [reply src/Avalonia.Themes.Default/Label.xaml:1]: found it. it was added after 0.9.12

**Comment by Gillibald** [comment]
Please soft reset your branch clean up everything and force push the result

**Comment by jmacato** [comment]
Have to close this since it hasn't been touched and fixed for a month. Also we object in adding redundant controls and im afraid that this `Label` control is one of them.

**Pull request closed without merging**